internal/loggers/pid_logger.go

```diff
@@ -48,7 +49,12 @@ func (vl *pidLogger) ExecutePID(header, mode, pid uint32, formula, protocol, nam
 	vl.logger.Debug().Msgf("formula is: %s", formula)
 	// todo: apply formula
 
-	_ = vl.storageQueue.Enqueue(resp.Value)
+	value, ok := resp.Value.(string)
```

Comment:
this feels odd - should we just have the Enqueue accept any? 
I mean I guess that implies the resulting JSON payload could have the signal value be a JSON string or JSON number. @KevinJoiner any thoughts on how flexible the schema types should be - or do we need two properties in the payload one for string type and another for numeric type. We can get on a call to explain more.

Reply:
> this feels odd - should we just have the Enqueue accept any?
>
This part I updated